app/src/androidTest/java/cmput301f17t13/com/catisadog/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package cmput301f17t13.com.catisadog;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumentation test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() throws Exception {
```

Comment:
If you're writing tests that use Android-isms (context and the like), check out Robolectric. With Robolectric you can write tests that use android components without needing to fully instrument the test.

Reply:
Link: http://robolectric.org

Interesting idea, @linster. Thanks for the recommendation :)

Reply:
I looked at the rubric and unfortunately we're forced to JUnit.